internal/handlers/pg/msg_find.go

```diff
@@ -95,7 +76,31 @@ func (h *Handler) MsgFind(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, er
 		}
 	}
 
-	fetchedDocs, err := h.fetch(ctx, db, collection)
```

Comment:
you've created a new `fetch` function abandoning the old one. Is there any reason for this?

Reply:
a comment parameter appeared and I created a new prototype function as a suggestion to replace the old one.. I should have mentioned it in the description of the PR. My bad, sorry